lib/ex_doc/markdown/hoedown.ex

```diff
@@ -12,10 +12,11 @@ defmodule ExDoc.Markdown.Hoedown do
 
   """
   def to_html(text, opts \\ []) do
-    Markdown.to_html(text,
-      autolink: Keyword.get(opts, :autolink, true),
-      fenced_code: Keyword.get(opts, :fenced_code, true),
-      tables: Keyword.get(opts, :tables, true))
+    text
+    |> Markdown.to_html(
+         autolink: Keyword.get(opts, :autolink, true),
+         fenced_code: Keyword.get(opts, :fenced_code, true),
+         tables: Keyword.get(opts, :tables, true))
```

Comment:
I will ident it only with two spaces.
you can read an interesting discussion about this https://github.com/elixir-lang/elixir/pull/3640#discussion-diff-41086777
read @whatyouhide's comment, i think it is a really neat way to address this indentention issue